adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/StructuredToAvroTransformer.java

```diff
@@ -48,8 +49,7 @@ public GenericRecord transform(StructuredRecord structuredRecord,
       String fieldName = field.name();
       co.cask.cdap.api.data.schema.Schema.Field schemaField = structuredRecordSchema.getField(fieldName);
       if (schemaField == null) {
-        throw new IllegalArgumentException("Input record does not contain the " + fieldName +
-                                             " field in output schema.");
+        throw new IllegalArgumentException("Input record does not contain the [" + fieldName + "] field.");
```

Comment:
didn't mean to use literal brackets. Just meant to say to keep this as it was before.